modules/gcp-gke/inputs.tf

```diff
@@ -40,6 +40,11 @@ variable "initial_node_count" {
   description = "Initial node count, per-zone for regional clusters."
 }
 
+variable "kubernetes_version" {
```

Comment:
Following up with the description of the PR.
The values here are not 'many' and do not change like extremely quickly either.

I am thinking, should we, a) either add some validation rules here where we consciously restrict the values that this can take, not to neglect the effort of maintaining the validation but since change versions of k8s is a fully-concious decision, that would make sense.

or b) define some `locals` with the possible values?

Reply:
Will do this in https://github.com/honestbank/api-cloud-infrastructure